packages/app/src/design-system/base/FormRadio.tsx

```diff
@@ -29,10 +29,22 @@ export const FormRadioGroup = ({
   ...rest
 }: FormRadioGroupProps) => {
   const arrayOfChildren = Children.toArray(children);
-  compatibleComponents(["FormRadioInput", "FormRadioLegend", "ValidationMessage"], arrayOfChildren);
-  const legend = arrayOfChildren.filter(child => (child as AuthorizedChildType).type.name === "FormRadioLegend");
-  const content = arrayOfChildren.filter(child => (child as AuthorizedChildType).type.name === "FormRadioInput");
-  const validation = arrayOfChildren.filter(child => (child as AuthorizedChildType).type.name === "ValidationMessage");
+  const acceptedComponentsNames = ["FormRadioLegend", "FormRadioInput", "ValidationMessage"];
+  compatibleComponents(acceptedComponentsNames, arrayOfChildren);
+  const [legend, radio, validation] =  arrayOfChildren.reduce(function (prev: Array<ReactNode>, next) {
```

Comment:
```typescript
const [legend, radio, validation] =  arrayOfChildren.reduce(function (prev: Array<ReactNode>, next) {
    let acceptedIdx: number = acceptedComponentsNames.findIndex(componentName => componentName === (next as AuthorizedChildType).type.name)
      prev[acceptedIdx] = [...(prev[acceptedIdx] || []), next]
    }
    ```

Reply:
J'ai essayé bien sûr mais avec Typescript ça passe pas, il faut typer. De plus le spread operator ne passe pas sur les Array of Object